deb/debian/install

```diff
@@ -1 +1 @@
-debian/collectd.conf opt/stackdriver/collectd/etc/
+debian/collectd.conf /etc/stackdriver/
```

Comment:
`etc/stackdriver/` (we need installation to work correctly even if the install prefix is specified).

Reply:
Done.